appwrite.template.json

```diff
@@ -10,6 +10,384 @@
+                },
+                {
+                    "key": "zip",
+                    "type": "string",
+                    "status": "available",
+                    "error": "",
+                    "required": false,
+                    "array": false,
+                    "size": 10,
+                    "default": null
+                },
+                {
+                    "key": "emails",
+                    "type": "string",
+                    "status": "available",
+                    "error": "",
+                    "required": true,
+                    "array": true,
+                    "format": "email",
+                    "default": null
+                }
+            ],
+            "indexes": []
+        },
+        {
+            "$id": "invoices",
+            "$permissions": [
+                "create(\"users\")"
+            ],
+            "databaseId": "default",
+            "name": "Invoices",
+            "enabled": true,
+            "documentSecurity": true,
+            "attributes": [
+                {
+                    "key": "userId",
+                    "type": "string",
+                    "status": "available",
+                    "error": "",
+                    "required": true,
+                    "array": false,
+                    "size": 50,
+                    "default": null
+                },
+                {
+                    "key": "clientId",
+                    "type": "string",
+                    "status": "available",
+                    "error": "",
+                    "required": true,
+                    "array": false,
+                    "size": 50,
+                    "default": null
+                },
+                {
+                    "key": "client",
+                    "type": "relationship",
+                    "status": "available",
+                    "error": "",
+                    "required": false,
+                    "array": false,
+                    "relatedCollection": "clients",
+                    "relationType": "manyToOne",
+                    "twoWay": false,
+                    "twoWayKey": "invoices",
+                    "onDelete": "cascade",
+                    "side": "parent"
+                },
+                {
+                    "key": "dueDate",
+                    "type": "datetime",
+                    "status": "available",
+                    "error": "",
+                    "required": true,
+                    "array": false,
+                    "format": "",
+                    "default": null
+                },
+                {
+                    "key": "products",
+                    "type": "string",
+                    "status": "available",
+                    "error": "",
+                    "required": false,
+                    "array": false,
+                    "size": 65535,
+                    "default": null
+                },
```

Comment:
This `products` attribute in `invoices` will be a stringified array of description, quantity, price, and total so that we can include all the data in the invoice faster than using relationships.